doc/source/guide/FAQ.rst

```diff
@@ -106,7 +106,34 @@ To troubleshoot your Lux installation, we recommend cloning `this repo <https://
 
 The Lux Jupyter widget does not show up when I print a dataframe.
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-  - If you recieve the output message :code:`LuxWidget(...)` but you do not see Lux widget show up, it is possible that the widget is not installed correctly. Run :code:`jupyter nbextension list` on the terminal, and you should see the following as one of the listed items. 
+To check if luxwidget is enabled, run the following code in your Jupyter Notebook/Lab:
+  
+  .. code-block:: bash
+  
+    import lux
+    lux.debug_info()
+  
+    
+  - If you receive the output message :code:`❌ IPython shell not available. Please note that Lux must be used within a notebook interface (e.g., Jupyter notebook, Jupyter Lab, JupyterHub, or VSCode`, please run your code in a notebook interface such as those mentioned in the message.
```

Comment:
Some of the lines are exceeding the docs page. You can preview the Sphinx docs by building it locally `sphinx-build -b html . build` and inspecting the HTML. 
![Screen Shot 2022-03-02 at 10 45 33 AM](https://user-images.githubusercontent.com/5554675/156427719-88497dbb-1002-4f6f-aa63-6262206823b0.png)